pkg/kubelet/container/container_gc.go

```diff
@@ -39,7 +39,7 @@ type ContainerGCPolicy struct {
 // Implementation is thread-compatible.
 type ContainerGC interface {
 	// Garbage collect containers.
-	GarbageCollect() error
+	GarbageCollect(allSourcesReady bool) error
```

Comment:
@ronnielai This is just a quick fix for this. :)
This is definitely not what we want, but we are in code freeze, we've to fix it with as few code as possible. :)